### DISCUSSION TITLE
Are we using older version of Catalog Items API ?

### DISCUSSION THREAD

**Discussion by jsaxdev** [abuzuhri/Amazon-SP-API-CSharp#88] [Q&A] [answered]
We are using Catalog Items API version: **v0**
Also there is other version of Catalog Items API version: **2020-12-01**

Which is latest ?
If **2020-12-01** is latest then maybe we need to use that version of Catalog Items API ?

**Comment by jsaxdev** [comment]
Found out in their API Migration Updates this:
**The deprecation of Selling Partner Catalog Items v0 operations will also occur on September 30, 2022.**

If we decide to leave **v0** for now then how we can name service and etc. for new version (**2020-12-01**) ?

**Reply by asifiq2000** [reply]
Hi dev,
I have to post the catalog on amazon using sp-api but I am new into this, can you help me how to make the rest request 
for catalog I have made an xml feed file but I could not post it so could you share a sample codes for a catalog feed posting
request. Thanks in advance.

**Comment by abuzuhri** [comment]
Hello @jsaxdev 

https://developer-docs.amazon.com/sp-api/
In the new documentation they don\t  mention it at all

**Comment by jsaxdev** [comment]
Hi @abuzuhri 
Take a look at this document:
https://developer-docs.amazon.com/sp-api/changelog/important-selling-partner-api-migration-updates-for-all-public-mws-developers

**Reply by abuzuhri** [reply]
Yes i know the migration date 

about which one the the newer its look like 2020-12-01 is latest 
https://github.com/amzn/selling-partner-api-docs/commits/main/references/catalog-items-api/catalogItems_2020-12-01.md

https://github.com/amzn/selling-partner-api-docs/commits/main/references/catalog-items-api/catalogItemsV0.md


but the question why its deleted from last doc https://developer-docs.amazon.com/sp-api

**Reply by abuzuhri** [reply]
I found it 
https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2020-12-01-reference

**Reply by abuzuhri** [reply]
we should work with this
https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2020-12-01-reference

I will add task to do this

**Reply by jsaxdev** [reply]
We can leave v0 version for now but I don't know how to name latest version in code.
Now we have for example **_CatalogItemService_** which is v0 and what to call newer version ?

**Comment by abuzuhri** [comment, marked as answer]
I think the true should be **CatalogItemServiceV20201201**
